docs/source/deployorderer/ordererplan.md

```diff
@@ -0,0 +1,84 @@
+
+Before proceeding with this topic, you should have reviewed the process for a Deploying a Certificate Authority (CA) for your organization in order to generate the identities and MSPs for the admins and ordering nodes in your organization. To learn how to use a CA to create these identities, check out [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html). Note that the best practice is to register and enroll a separate node identity for each ordering node and to use distinct TLS certificates for each node.
+
+Note that the `cryptogen` tool should never be used to generate any identities in a production scenario.
+
+In this deployment guide, we’ll assume that all ordering nodes will be created and owned by the same orderer organization. However, it is possible for multiple organizations to contribute nodes to an ordering service, both during the creation of the ordering service and after the ordering service has been created.
+
+## Folder management
+
+While it is possible to bootstrap an ordering node using a number of folder structures for your MSPs and certificates, we do recommend the folder structure outlined in [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html#decide-on-the-structure-of-your-folders-and-certificates) for the sake of consistency and repeatability. Although it is not required, these instructions will presume that you have used that folder structure.
+
+## Certificates from a non-Fabric CA
+
+While it is possible to use a non-Fabric CA to generate identities, this process requires that you manually construct the MSP folders the ordering service and its organization need. That process will not be covered here and will instead focus on using a Fabric CA to generate the identities and MSP folders for you.
+
+## Transport Layer Security (TLS) enablement
+
+To prevent “man in the middle” attacks and otherwise secure communications, the use of TLS is a requirement for any production network. Therefore, in addition to registering your ordering nodes identities with your organization CA, you will also need to create certificates for your ordering nodes with the TLS CA of your organization. These TLS certificates will be used by the ordering nodes when communicating with the network.
+
+## Creating the system channel genesis block
+
+Note: “consenters” refers to the nodes servicing a particular channel at a particular time. For each channel, the “consenters” may be a subset of the ordering nodes available in the system channel.
+
+Every ordering node must be bootstrapped with a configuration block, known as the "genesis block", that contains configuration of a channel. While application channels also contain genesis blocks, the genesis block an ordering node is bootstrapped with is the genesis block of what’s called the “orderer system channel”.
```

Comment:
Doesn't have to be genesis block, can also be a later configuration block (note, this is why the orderer.yaml config option name changed from 'GenesisFile' to 'BootstrapFile' starting in v2.0.

```suggestion
Every ordering node must be bootstrapped with a configuration block from the system channel, either the system channel "genesis block" or a later configuration block. This guide will assume you are creating a new ordering service and will therefore bootstrap ordering nodes from a system channel genesis block.
```